tests/test_client.py

```diff
@@ -76,7 +76,7 @@ def get_all(self, stream, start_date=None, end_date=None):
             return [obj for page, _ in self.get_refunds(start_date, None) for obj in page]
         elif stream == 'payments':
             if not self.PAYMENTS:
-                return [obj for page, _ in self.get_payments(start_date, None) for obj in page]
+                self.PAYMENTS += [obj for page, _ in self.get_payments(start_date, None) for obj in page]
```

Comment:
Since the above line is if not self.PAYMENTS it would be empty. So this should just be self.PAYMENTS =, not += right?

Reply:
It can be either I guess, since it is instantiated to an empty string. Equals probably makes more sense.

Reply:
* empty list

Reply:
not string

Reply:
I know it could be either, but I think it's confusing because the reader of the code has to stop and think for a second about what that might mean...

Reply:
Changed.